pkg/clusteragent/autoscaling/workload/controller_horizontal.go

```diff
@@ -133,13 +134,15 @@ func (hr *horizontalController) performScaling(ctx context.Context, podAutoscale
 	if err != nil {
 		err = fmt.Errorf("failed to scale target: %s/%s to %d replicas, err: %w", scale.Namespace, scale.Name, horizontalAction.ToReplicas, err)
 		hr.eventRecorder.Event(podAutoscaler, corev1.EventTypeWarning, model.FailedScaleEventReason, err.Error())
+		telemetryHorizonalScale.Inc(scale.Namespace, scale.Name, strconv.Itoa(int(horizontalAction.ToReplicas)), strconv.Itoa(int(horizontalAction.FromReplicas)), strconv.Itoa(int(*horizontalAction.RecommendedReplicas)), "true")
```

Comment:
For values, it's easier to expose a `Gauge` with the `ToReplicas` value (basically seeing history). We can counters for number of success/failures, not tracking the `To/From` (not needed). It's interesting to have the target `namespace` and `name` but we also want the `DatadogPodAutoscaler` namespace and name (`namespace` is the same in both cases and could be omitted)